vagrant/provisioning/roles/arkcase-post-upgrade-start-config/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+- name: start config server
+  become: yes
+  systemd:
+    name: "config-server"
+    state: started
+
+- name: wait for config server
```

Comment:
I don't understand the name of this role, since the upgrade plays already start both config server and arkcase, see roles/arkcase-post-upgrade/tasks/main.yml.  

Also you should remove this code from roles/arkcase-post-upgrade/tasks/main.yml, so as to avoid code duplication, and then call this role from all the plays where you want to start ArkCase after initial installation (arkcase-dev.yml, all the various arkcase-ee-... roles, basically any play except for arkcase-ce.yml.

Reply:
Also of course add this role to the various upgrade plays, since you will have removed it from roles/arkcase-post-upgrade/tasks/main.yml.

Reply:
And consider renaming the role itself to something like 'start-arkcase', which describes what it does, versus when it is called.